src/components/menus/branch-menu.js

```diff
@@ -0,0 +1,337 @@
+                />
+                {equipmentType === equipments.lines && (
+                    <MenuItem
+                        className={classes.menuItem}
+                        onClick={() => handleLockout()}
+                        disabled={
+                            !isNodeEditable ||
+                            branch.branchStatus === 'PLANNED_OUTAGE'
+                        }
+                    >
+                        <ListItemIcon>
+                            <LockOutlinedIcon />
+                        </ListItemIcon>
+
+                        <ListItemText
+                            className={classes.listItemText}
+                            primary={
+                                <Typography noWrap>
+                                    {intl.formatMessage({
+                                        id: getTranslationKey('Lockout'),
+                                    })}
+                                </Typography>
+                            }
+                        />
+                    </MenuItem>
+                )}
+                <MenuItem
+                    className={classes.menuItem}
+                    onClick={() => handleTrip()}
+                    disabled={
+                        !isNodeEditable ||
+                        branch.branchStatus === 'FORCED_OUTAGE'
+                    }
+                >
+                    <ListItemIcon>
+                        <OfflineBoltOutlinedIcon />
+                    </ListItemIcon>
+
+                    <ListItemText
+                        className={classes.listItemText}
+                        primary={
+                            <Typography noWrap>
+                                {intl.formatMessage({
+                                    id: getTranslationKey('Trip'),
+                                })}
+                            </Typography>
+                        }
+                    />
+                </MenuItem>
+                {equipmentType === equipments.lines && (
+                    <MenuItem
+                        className={classes.menuItem}
+                        onClick={() => handleEnergise('ONE')}
+                        disabled={
+                            !isNodeEditable ||
+                            (branch.terminal1Connected &&
+                                !branch.terminal2Connected)
+                        }
+                    >
+                        <ListItemIcon>
+                            <EnergiseOneSideIcon />
+                        </ListItemIcon>
+
+                        <ListItemText
+                            className={classes.listItemText}
+                            primary={
+                                <Typography noWrap>
+                                    {intl.formatMessage(
+                                        {
+                                            id: getTranslationKey(
```

Comment:
We can keep the key 'EnergiseOnOneEnd' with the 'Line' at the end nope ?

Reply:
I assume one day we'll have "Energize from transformer" so I use prefixes and getTranslationKey everywhere in this menu.